core/queryalgebra/model/src/main/java/org/eclipse/rdf4j/query/algebra/UnaryValueOperator.java

```diff
@@ -10,6 +10,8 @@
  *******************************************************************************/
 package org.eclipse.rdf4j.query.algebra;
 
+import java.util.Objects;
+
```

Comment:
If you look at the javadoc, this is now not consistent with the implementation. Maybe you want to revisit?

```
An abstract superclass for unary value operators which, by definition, has one argument.
```

At least for the `Count` node this is not always the case.